### PR TITLE
[New Rule] Package kea removed

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

* New Rule Package kea removed
* Add `package_kea_removed` to RHEL 10 profiles
   * Exclusion in other profiles.

#### Rationale:

Kea is the DHCP server in RHEL 10 as the ISC DHCP server is no longer supported.

### CODE REVIEW

File: components/dhcp.yml [hunk 1]
```diff
@@ -20,3 +20,4 @@ rules:
 - package_dhcp_removed
 - service_dhcpd_disabled
 - sysconfig_networking_bootproto_ifcfg
+- package_kea_removed
```
Comment (jan-cerny) [inline components/dhcp.yml:23]: Does the kea package replace the dhcp package? Is the dhcp package present in RHEL 10? If the dhcp package isn't present in RHEL 10 we should remove the rule package_sshd_removed from RHEL 10 content.

Comment (Mab879) [reply components/dhcp.yml:23]: It does, I have removed `package_dhcp_removed` from all profiles in RHEL 10.

**Review by Xeicker — APPROVED** [review]
changes LGTM

**Review by jan-cerny — APPROVED** [review]
I have verified that the rule package_kea_removed is present in the built RHEL 10 DS and that the rule package_dhcp_removed isn't present there.

**Comment by jan-cerny** [comment]
@teacup-on-rockingchair please, take a look

**Review by teacup-on-rockingchair — APPROVED** [review]
👍

**Pull request merged**